sdk/core/azure-core/src/http/curl/curl_connection_private.hpp

```diff
@@ -31,19 +31,19 @@
 typedef struct x509_store_ctx_st X509_STORE_CTX;
 
 namespace Azure { namespace Core {
-  namespace _internal {
+  namespace _detail {
```

Comment:
Same comment: Should this be in Http?

Reply:
Same comment as above - organizationally there could be sense in doing that, technically, it is not important - unlike the fact that it is important that every lib does declare its specializations in its OWN namespace. But when it is within the same library - there is no technical problem (potential linker error due to duplicate symbols) that moving it to different namespace is solving; it is only the aesthetics.
I'd say it is a bit more hassle to re-declare UniqueHandle and UniqueHandleHelper in multiple namespaces per package, I'd say it is not worth it. But it totally can be done.